src/components/ebay-date-textbox/component.js

```diff
@@ -66,10 +66,12 @@ export default class extends Marko.Component {
     }
 
     calculateNumMonths() {
-        this.state.numMonths =
-            document.documentElement.clientWidth < MIN_WIDTH_FOR_DOUBLE_PANE
-                ? 1
-                : 2;
+        if (document !== undefined) {
```

Comment:
This check won't work, `document` would be undefined in server side. You might have to change this to
```
typeof document === 'undefined'
```

Reply:
Can't we just check whether `document` is falsy? And to keep conditionals shorter and the code easier to read, it's usually better to exit functions as quickly as possible...

`````js
if (!document) {
  return;
}
...
`````

Reply:
Yes we need to check for typeof. This can throw a compile error if document is undefined.

Reply:
Thanks for the input everyone, I've updated it now